tasks/graylog_post.yml

```diff
@@ -10,6 +10,17 @@
     graylog_elasticsearch_host_list: "{{ graylog_elasticsearch_facts_output.results | map(attribute='ansible_facts.graylog_elasticsearch_facts') | list }}"
   when: graylog_elasticsearch_facts_output.results is defined
 
+- name: Set fact variable for MongoDB list each items
+  set_fact:
+    graylog_mongodb_facts: "{{ item }}"
+  with_items: "{{ graylog_mongodb_hosts }}"
+  register: graylog_mongodb_facts_output
+
+- name: Set fact variable for all host MongoDB
+  set_fact:
+    graylog_mongodb_host_list: "{{ graylog_mongodb_facts_output.results | map(attribute='ansible_facts.graylog_mongodb_facts') | list }}"
```

Comment:
may need to change this logic if above list is changed

Reply:
further review see that graylog_mongodb_host_list have the same value as graylog_mongodb_hosts

maybe unneccessary step?